### PR TITLE
Change paid to paidAmount

### PR DESCRIPTION
**Pull request by DennieDan**
All instances have been changed
Please help approve this.
Then we can merge to the master.
Any of our team members have started your v1.4, please help merge to your current working branch to prevent tedious merge conflicts resolvement.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ Examples:
 
 ### Showing income data: `income`
 
-Shows the total amount of tuition fee you have received from the students and the amount that hasn't been paid.
+Shows the total amount of tuition fee you have received from the students and the amount that hasn't been paidAmount.
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:123]: This might have been better if paid was not changed to paidAmount

File: docs/UserGuide.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ Format: `help`
 
 Adds a student to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT f/FEE [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT f/FEE [paidAmount/PAID_AMOUNT] [owed/OWED_AMOUNT]`
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:79]: Would owed also need to be changed to owedAmount?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT f/FEE
 </box>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 t/Sunday-1000-1200 s/Geography f/100 paid/100 owed/0`
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 t/Sunday-1000-1200 s/Geography f/100 paidAmount/100 owed/0`
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:98]: Would owed also need to be changed to owedAmount?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -208,7 +208,7 @@ _Details coming soon ..._
 
 Action     | Format, Examples
 -----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
-**Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT f/FEE [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/Monday-0800-1000 s/GP f/300 paid/300`
+**Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT f/FEE [paidAmount/PAID_AMOUNT] [owed/OWED_AMOUNT]` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/Monday-0800-1000 s/GP f/300 paidAmount/300`
```
Comment (Soumyadip-cmd) [inline docs/UserGuide.md:211]: Would owed also need to be changed to owedAmount?

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 5]
```diff
@@ -13,6 +13,6 @@ public class CliSyntax {
     public static final Prefix PREFIX_SCHEDULE = new Prefix("t/");
     public static final Prefix PREFIX_SUBJECT = new Prefix("s/");
     public static final Prefix PREFIX_RATE = new Prefix("r/");
-    public static final Prefix PREFIX_PAID = new Prefix("paid/");
+    public static final Prefix PREFIX_PAID_AMOUNT = new Prefix("paid/");
```
Comment (Soumyadip-cmd) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:16]: Should the prefix be paid or paidAmount? In the user guide the prefix was changed to paidAmount

File: src/main/java/seedu/address/ui/StudentCard.java [hunk 6]
```diff
@@ -62,8 +62,9 @@ public StudentCard(Student student, int displayedIndex) {
         address.setText(student.getAddress().value);
         email.setText(student.getEmail().value);
         subjectAndSchedule.setText(student.getSubject().toString() + ": " + student.getSchedule().toString());
-        rateAndPaidAndOwedAmount.setText("Rate: $" + student.getRate().toString() + "/h, "
-            + "Paid: $" + student.getPaid().toString() + ", " + "Owed: $" + student.getOwedAmount().toString());
+        rateAndPaidAmountAndOwedAmount.setText("Rate: $" + student.getRate().toString() + "/h, "
+            + "PaidAmount: $" + student.getPaidAmount().toString() + ", "
+                + "Owed: $" + student.getOwedAmount().toString());
```
Comment (Soumyadip-cmd) [inline src/main/java/seedu/address/ui/StudentCard.java:67]: Same as above. Would owed also need to be changed to owedAmount?

**Review by TY1Fan — APPROVED** [review]
LGTM!

File: src/test/java/seedu/address/storage/JsonAdaptedStudentTest.java [hunk 7]
```diff
@@ -26,7 +26,7 @@ public class JsonAdaptedStudentTest {
     private static final String INVALID_SCHEDULE = "Anyday 7pm-9pm";
     private static final String INVALID_SUBJECT = "PHYSICAL EDUCATION";
     private static final String INVALID_RATE = "123.23/h";
-    private static final String INVALID_PAID = " ";
+    private static final String INVALID_PAID_AMOUNT = " ";
```
Comment (Soumyadip-cmd) [inline src/test/java/seedu/address/storage/JsonAdaptedStudentTest.java:29]: Good that you changed the constant

**Review by Soumyadip-cmd — COMMENTED** [review]
Overall is good. Might need to confirm if prefix will be paid or paidAmount

**Comment by DennieDan** [comment]
> Overall is good. Might need to confirm if prefix will be paid or paidAmount

For user convenience, I think prefix should be kept to `/paid` and `/owed` for short

File: docs/UserGuide.md (hunk 4 above)
Comment (DennieDan) [reply docs/UserGuide.md:211]: Oops, thanks for pointing this out

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM

**Review by teamotitan — APPROVED** [review]
LGTM

**Pull request merged**